cmd/backup/script.go

```diff
@@ -159,6 +164,45 @@ func newScript() (*script, error) {
 		}
 	}
 
+	if s.c.SSHHostName != "" {
+		var authMethods []ssh.AuthMethod
+
+		if s.c.SSHPassword != "" {
+			authMethods = append(authMethods, ssh.Password(s.c.SSHPassword))
+		}
+
+		if _, err := os.Stat(s.c.SSHIdentityFile); err == nil {
+			key, err := ioutil.ReadFile(s.c.SSHIdentityFile)
+			if err == nil {
+				var signer ssh.Signer
+				if s.c.SSHPassphrase != "" {
+					signer, err = ssh.ParsePrivateKeyWithPassphrase(key, []byte(s.c.SSHPassphrase))
+				} else {
+					signer, err = ssh.ParsePrivateKey(key)
+				}
+				if err == nil {
+					authMethods = append(authMethods, ssh.PublicKeys(signer))
+				}
+			}
+		}
+
+		sshClient, err := ssh.Dial("tcp", fmt.Sprintf("%v:%v", s.c.SSHHostName, s.c.SSHPort), &ssh.ClientConfig{User: s.c.SSHUser, Auth: authMethods, HostKeyCallback: ssh.InsecureIgnoreHostKey()})
```

Comment:
Two nitpicks:
- can we break this into multiple lines, it's very long
- can we use `%s` verbs in the `fmt.Sprintf` calls? We do know we are passing strings and don't need magic reflection under the hood to figure out how to stringify these values